misc/python/materialize/zippy/balancerd_capabilities.py

```diff
@@ -0,0 +1,14 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+from materialize.zippy.framework import Capability
+
+
+class BalancerdIsRunning(Capability):
+    pass
```

Comment:
Do we need a TODO here?

Reply:
Nope, this object has no methods or anything, it is used as a marker that balancerd is running and can be stopped, or not running and can be started. Same as `MzIsRunning`.

Reply:
Ok 👍